### ISSUE TITLE
Nonsensical zpool status output during scrub

### ISSUE THREAD

**Issue by tc1415** [zfsonlinux/zfs#8891]
### System information

Type | Version/Name
 --- | --- 
Distribution Name	|  Fedora
Distribution Version	|  30
Linux Kernel	|  5.1.7
Architecture	|  x86_64
ZFS Version	|  0.8.0
SPL Version	|  -

### Describe the problem you're observing
After setting a scrub running and waiting a bit, zpool status produces nonsensical output reporting it has scanned many TB of data on a pool with only 212 GB of data in it.

```  pool: pool
 state: ONLINE
  scan: scrub in progress since Wed Jun 12 14:12:54 2019
        212G scanned at 325M/s, 245T issued at 375G/s, 212G total
        0B repaired, 118068.85% done, no estimated completion time
config:

        NAME              STATE     READ WRITE CKSUM
        pool              ONLINE       0     0     0
          encrypted_nvme  ONLINE       0     0     0

errors: No known data errors
```

The man page notes that it can "slightly" exceed 100 % but this is three orders-of-magnitude higher!

### Describe how to reproduce the problem
`zpool scrub pool`
Wait a bit (or do `watch zpool status pool` and watch it)
`zpool status pool`
Observe output.

**Comment by DeHackEd** [comment]
Already fixed in #8766, will be 0.8.1

**Comment by drescherjm** [comment]
I was looking for that but could not find it in a quick search.

**Comment by tc1415** [comment]
Oh, excellent. I presume the issue is entirely cosmetic and the underlying scrub operation does what it is supposed to? If so I'll just ignore it and wait for 0.8.1 :-)

**Comment by behlendorf** [comment]
@tc1415 that's right.  It's entirely cosmetic and the underlying scrub does correctly verify everything.

**Comment by tc1415** [comment]
Confirmed fixed in 0.8.1, so closing.
